server/clcachesrv.py

```diff
@@ -36,8 +36,11 @@ def getFileHash(self, path):
         return hashsum
 
     def _startWatching(self, dirname):
-        ev = pyuv.fs.FSEvent(self._loop)
-        ev.start(dirname, 0, self._onPathChange)
+        self.ev = pyuv.fs.FSEvent(self._loop)
+        self.ev.start(dirname, 0, self._onPathChange)
+        
+    def __del__(self):
+        self.ev.stop()
```

Comment:
This patch seems to be worth it's own PR, i.e. it appears to be a bit out of scope for the memcached support, doesn't it?